docs/manuals/man8/pkispawn.8.md

```diff
@@ -1441,28 +1447,28 @@ $ systemctl status pki-tomcatd@<pki_instance_name>.service
 To obtain a detailed status of a Tomcat PKI instance named &lt;pki_instance_name&gt;:
 
 ```
-$ pkidaemon status tomcat <pki_instance_name>
+$ pki-server status <pki_instance_name>
 ```
 
 To obtain a detailed status of all Tomcat PKI instances:
 
 ```
-$ pkidaemon status tomcat
+$ pki-server status
```

Comment:
I think this part can be removed. `pki-server status` will only work with one instance at a time.